### PR TITLE
Kolya Venturi Morse Code Implementation

### PR DESCRIPTION
**Pull request by kolyaventuri**

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require 'rake/testtask'
+
+Rake::TestTask.new do |t|
```
Comment (mikedao) [inline Rakefile]: 👏

File: lib/translator.rb [hunk 2]
```diff
@@ -40,4 +40,43 @@ def initialize
                     "0" => "-----",
                     " " => " "}
   end
+
+  def eng_to_morse(text)
+    letters = text.downcase.split("")
+    morse_chars = letters.map do | char |
+      morse_char = @dictionary[char]
```
Comment (mikedao) [inline lib/translator.rb]: You could just leave this as `@dictionary.char` and not do a variable assignment and explicit return

File: lib/translator.rb [hunk 3]
```diff
@@ -40,4 +40,43 @@ def initialize
...
+      alpha_num_char.join("")
+    end
+
+    alpha_numeric_words.join(" ")
+  end
+
+
+  def from_file(filename)
+    if(!File.exist?(filename))
+      puts "File does not exist!"
```
Comment (mikedao) [inline lib/translator.rb]: Good looking for the edge case. You could also use guard statement here.

File: test/translator_test.rb [hunk 4]
```diff
@@ -1 +1,35 @@
-# YOUR TESTS GO IN HERE!
\ No newline at end of file
+require_relative 'test_helper.rb'
```
Comment (mikedao) [inline test/translator_test.rb]: Good use of `test_helper`

File: .gitignore [hunk 5]
```diff
@@ -0,0 +1,106 @@
+# Git #
```
Comment (mikedao) [inline .gitignore]: Better to set a standard global .gitignore than a separate one here.
